src/odemis/acq/fastem.py

```diff
@@ -995,28 +995,22 @@ def run(self, f, stream, stage, area, live_stream):
         overlap = STAGE_PRECISION / stream.emitter.horizontalFoV.value
         logging.debug("Overlap is %s%%", overlap * 100)  # normally < 1%
 
-        def _pass_future_progress(sub_f, start, end):
-            f.set_progress(start, end)
-
         # Note, for debugging, it's possible to keep the intermediary tiles with log_path="./tile.ome.tiff"
-        self.sf = stitching.acquireTiledArea([stream], stage, area, overlap, registrar=REGISTER_IDENTITY,
+        self._sub_future = stitching.acquireTiledArea([stream], stage, area, overlap, registrar=REGISTER_IDENTITY,
                                         focusing_method=FocusingMethod.NONE)
-        # Connect the progress of the underlying future to the main future
-        # FIXME removed to provide better progress update in GUI
-        #  When _tiledacq.py has proper time update implemented, add this line here again.
-        # sf.add_update_callback(_pass_future_progress)
```

Comment:
It's worth to keep. Eventually that should be the right thing to do. So let's keep track of it. 
It just needs to keep the a reference to the `future` in `__init__()`, so that this is still possible.